### PR TITLE
Tickets/dm 8947

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

File: python/lsst/ctrl/orca/DatabaseConfigurator.py [hunk 1]
```diff
@@ -22,35 +22,35 @@
...
+    @param databaseConfig - Config
+        database configuration object
+    @param prodConfig - Config
+        production configuration object
 
-        # run id
-        self.runid = runid
+    Note: this is currently unused, but being kept in place for when
+    orchestration records provenance for runs
+    """
```
Comment (mxk62) [inline python/lsst/ctrl/orca/DatabaseConfigurator.py:48]: To be honest, I'm not sure if it is a must, but the Developer Guide seems to prefer [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) format of docstrings.

```python
class DatabaseConfigurator(object):
    """Configures a database for use.

    Parameters
    ----------
    runid : `str`
        Run indetfier.
    ...

    Note
    ----
    ...
    """
```

File: python/lsst/ctrl/orca/DatabaseConfigurator.py [hunk 2]
```diff
@@ -22,35 +22,35 @@
...
-        # call setup on the configurator object
-
-        # if provSetup is not None:
-        #     may call provSetup.addProductionRecorder(ProvenanceRecorder)
-        #     may call provSetup.addWorkflowRecordCmd(string)
-
+    def setup(self, provSetup):
+        """setup for a new ru
+        @param provSetup - provenance object
+        """
```
Comment (mxk62) [inline python/lsst/ctrl/orca/DatabaseConfigurator.py:63]: See the comment above.

File: python/lsst/ctrl/orca/DatabaseConfigurator.py (hunk 1 above)
Comment (srp3rd) [reply python/lsst/ctrl/orca/DatabaseConfigurator.py:48]: I fixed this.

File: python/lsst/ctrl/orca/DatabaseConfigurator.py (hunk 2 above)
Comment (srp3rd) [reply python/lsst/ctrl/orca/DatabaseConfigurator.py:63]: I fixed this.

**Pull request merged**